guides/common/modules/proc_migrating-project-or-smartproxy-using-backup.adoc

```diff
@@ -0,0 +1,50 @@
+[id="migrating-{project-context}-or-proxy-using-backup_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9
+
+You can migrate your existing {ProjectServer} and {SmartProxyServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+The migration involves creating a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then restore or clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You can use cloning for {ProjectServer} backups but not for {SmartProxyServer} backups.
+====
+endif::[]
+ifndef::satellite[]
+The migration involves creating a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then restore on the new {EL}{nbsp}9 system.
+endif::[]
+
+.Procedure
+. Perform a full backup of your {ProjectServer} or {SmartProxy}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer} or {SmartProxyServer}] in _{AdministeringDocTitle}_.
+. Deploy a system with {EL}{nbsp}9.
+This is the target server.
+ifdef::satellite[]
+The {ProjectServer} backup can be cloned to the target without installing {Project} on the target server.
+If you plan to restore the backup on the target server, install {ProjectServer} or {SmartProxyServer} on the target {EL}{nbsp}  system:
+endif::[]
+ifndef::satellite[]
+Install {ProjectServer} or {SmartProxyServer} on the target {EL}{nbsp}9 system:
+endif::[]
+ifeval::[ "{mode}" == "connected" ]
+** To install {ProjectServer}, see {InstallingServerDocURL}[_{InstallingServerDocTitle}_].
+endif::[]
+ifeval::[ "{mode}" == "disconnected" ]
+** To install {ProjectServer}, see {InstallingServerDisconnectedDocURL}[_{InstallingServerDisconnectedDocTitle}_].
+endif::[]
+** To install a {SmartProxyServer}, see {InstallingSmartProxyDocURL}[_{InstallingSmartProxyDocTitle}_].
+ifdef::satellite[]
+. Restore the {ProjectServer} backup by following one of these options:
+** Restore the backup.
+Restore does not significantly alter the source system and requires additional configuration.
+For more information, see {AdministeringDocURL}restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a backup] in _{AdministeringDocTitle}_.
+** Clone the server.
+Clone configures hostname for the target server and you can test it before retiring the source server.
+For more information, see {AdministeringDocURL}cloning_{project-context}_server#sec-Cloning_to_Target[Cloning to the target server] in _{AdministeringDocTitle}_.
```

Comment:
```suggestion
For more information, see {AdministeringDocURL}#sec-Cloning_to_Target[Cloning to the target server] in _{AdministeringDocTitle}_.
```
Otherwise, the link won't work.